views/admin.functions.setup.options.php

```diff
@@ -144,6 +144,41 @@ function sailthru_initialize_setup_options() {
 			);
 		}
 
+		add_settings_section(
+			'recaptcha_setup_section',   // ID used to identify this section and with which to register options
+			__( '', 'sailthru-for-wordpress' ),    // Title to be displayed on the administration page
+			'recaptcha_setup_callback',   // Callback used to render the description of the section
+			'sailthru_setup_options'   // Page on which to add this section of options
+		);
+	
+		add_settings_field(
+			'google_recaptcha_site_key',
+			__( 'reCaptcha Site Key', 'sailthru-for-wordpress' ),
+			'sailthru_html_text_input_callback',
+			'sailthru_setup_options',
+			'recaptcha_setup_section',
+			array(
+				'sailthru_setup_options',
+				'google_recaptcha_site_key',
+				'',
+				'google_recaptcha_site_key',
+			)
+		);
+	
+		add_settings_field(
+			'google_recaptcha_secret',
+			__( 'reCaptcha Secret Key', 'sailthru-for-wordpress' ),
+			'sailthru_html_text_input_callback',
```

Comment:
If you created a new callback, would that give you ability to make this a password type input?